src/node_builtins.cc

```diff
@@ -688,48 +690,44 @@ void BuiltinLoader::HasCachedBuiltins(const FunctionCallbackInfo<Value>& args) {
 // TODO(joyeecheung): It is somewhat confusing that Class::Initialize
 // is used to initialize to the binding, but it is the current convention.
 // Rename this across the code base to something that makes more sense.
+void BuiltinLoader::Initialize(IsolateData* isolate_data,
+                               Local<FunctionTemplate> target) {
```

Comment:
I am not sure if creating one FunctionTemplate for each binding is going to work for all the bindings - suppose they have many more templates for different BaseObjects in one binding?

Reply:
We can attach `FunctionTemplate`s to the binding's target `FunctionTemplate`, just like the `exports` object. I've added a new commit e07122b8b689e85fc2ecfef28bbc279431112584 to show how this is achieved in the worker binding.

Reply:
I see, that's a clever trick 👍